server/routes/companyRoutes.js

```diff
@@ -63,6 +63,23 @@ router.get('/profile/:companyId', (req, res) => {
     })
 })
 
+router.post('/profile/:id', (req,res) => {
+  //grabs id from the url
+  const id = req.params.id
+  const body = req.body
```

Comment:
This line really isn't adding any value. Perhaps if we at least call the variable `profile` it will add some semantic value.

Reply:
removed the comment